etl/assets.py

```diff
@@ -101,33 +106,38 @@ def wikipedia_anti_recommendations(
     documents_of_wikipedia_articles_with_summaries: DocumentTuple,
     openai_settings: OpenaiSettings,
     output_config: OutputConfig,
-) -> AntiRecommendationKeysByKeyTuple:
+) -> AntiRecommendationGraphTuple:
     """Materialize an asset of Wikipedia anti-recommendations."""
 
     wikipedia_anti_recommendations_embedding_store = OpenaiEmbeddingPipeline(
         openai_settings=openai_settings,
         output_config=output_config,
-    ).create_embedding_store(documents_of_wikipedia_articles_with_summaries.documents)
+    ).create_embedding_store(
+        documents=documents_of_wikipedia_articles_with_summaries.documents,
+        distance_strategy=DistanceStrategy.COSINE,
```

Comment:
Nit: these should probably come from settings (with defaults) rather than hard-coding them. Don't worry about it in this PR.